petstore-api-proxy-from-github/readme.md

```diff
@@ -0,0 +1,52 @@
+# Petstore API Proxy from GitHub Repository
+
+## Use case
+
+This sample demonstrates how to create a proxy for a Petstore API using a GitHub repository. The API Proxy component will forward requests to the actual Petstore API, allowing you to manage and monitor the API traffic.
+
+The Petstore API provides endpoints to manage pet information, store inventory, and user data. By creating a proxy for this API, you can add additional layers of security, rate limiting, and analytics.
+
+The following steps will guide you through the process of creating and deploying the Petstore API Proxy component in Choreo:
+
+1. **Create the API Proxy**: Create the API Proxy using the provided sample values.
+2. **Configure and Deploy the API Proxy**: Configure the necessary settings and deploy the API proxy to make it available for use.
```

Comment:
API Proxy or API proxy ? 

I think it should be in lowercase

Reply:
Yes it looks better. Fixed & thanks